service/src/main/java/tds/session/web/endpoints/SessionController.java

```diff
@@ -50,6 +51,12 @@ public SessionController(final SessionService sessionService) {
         return ResponseEntity.ok(session);
     }
 
+    @RequestMapping(value = "/", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I think this should drop the `/` and just be `http://host:port/sessions?sessionIds=`.